protocol/x/clob/keeper/process_operations.go

```diff
@@ -38,6 +57,26 @@ func (k Keeper) ProcessProposerOperations(
 		return errorsmod.Wrapf(types.ErrInvalidMsgProposedOperations, "Error: %+v", err)
 	}
 
+	// If grpc streams are on, send absolute fill amounts from local + proposed opqueue to the grpc stream.
+	// This must be sent out to account for checkState being discarded and deliverState being used.
+	if streamingManager := k.GetGrpcStreamingManager(); streamingManager.Enabled() {
+		localValidatorOperationsQueue, _ := k.MemClob.GetOperationsToReplay(ctx)
```

Comment:
so one issue that I just thought of is we are probably going to enable optimistic block execution in the near term, which means that the whole FinalizedBlock will be done in a separate go routine. Accessing `k.MemClob` here (and in other deliver state ABCI callbacks) is dangerous and can potentially lead to race conditions. 

are there any alternatives here?

Reply:
we can merge to unblock first, but should prob keep this in mind. grpc streams and OE might not co-exist